requirements-dev.txt

```diff
@@ -3,6 +3,7 @@ coverage<5.0
 graphviz>=0.8.3
 ipython
 Keras>=2.0.8
+matplotlib<=3.1.0
```

Comment:
it should also be in the non-dev `requirements.txt`

Reply:
Ok, so it means the problem is not resolved 😕 
I'll see if I come up with any idea 😅